### ISSUE TITLE
GET /api/organizations does not return a paginated list.

### ISSUE THREAD

**Issue by k1won** [opencv/cvat#4430]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

- The /api/organizations GET method returns a paginated list of organizations according to query parameters. (reference : api doc)
### Current Behaviour

- According to the api documentation, /api/organizations is specified to return a paginated list, but the execution result is not.
- The pagination_class attribute value of OrganizationViewSet class is set to None.
```python
class OrganizationViewSet(viewsets.ModelViewSet):
    queryset = Organization.objects.all()
    search_fields = ('name', 'owner')
    filter_fields = list(search_fields) + ['id', 'slug']
    lookup_fields = {'owner': 'owner__username'}
    ordering_fields = filter_fields
    ordering = '-id'
    http_method_names = ['get', 'post', 'patch', 'delete', 'head', 'options']
    pagination_class = None
    iam_organization_field = None
```
### Possible Solution

- remove or comment out the "pagination_class=None" line.
### Steps to Reproduce (for bugs)

1. execute the GET /api/organizations

### Your Environment

- Git hash commit (`git log -1`): b0e6074c
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS): ubuntu 18.04
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by nmanovic** [comment]
@k1won , thanks for the feedback. We make the decision in the past but now I think it is better to unify the interface and return pagination list for organizations.

@bsekachev , @Marishka17 , what do you think?

**Comment by bsekachev** [comment]
@nmanovic 

I do not see significant issues, but in this case need to add an ability to get organization by slug (using search, or filtering, or somehow else).

**Comment by k1won** [comment]
@bsekachev , @nmanovic ,
I agree with that there are no serious problems.
However, I would like to point out the fact that the actual behavior differs from what is stated in the api documentation.
If you don't want to change the implementation, it's better to change the descriptions of the api documentation I think.
Additionally, one more thing to consider is that most other list methods were implemented to return a pagenated list.
Whatever we choose,  it's better to reduce unnecessary confusion.

**Comment by nmanovic** [comment]
@k1won , I agree. I also think that we need to reduce unnecessary confusion. Need to change the implementation.

**Comment by zhiltsov-max** [comment]
Fixed in #5557
